src/OrchardCore.Build/OrchardCore.Commons.props

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <LangVersion>10.0</LangVersion>
-    <VersionPrefix>1.7.0</VersionPrefix>
+    <VersionPrefix>1.7.1</VersionPrefix>
```

Comment:
@sebastienros this may not be needed here, right? 

If we take this and merge it into main then we would have to update main all over again to `1.8.0`

Reply:
This is required, but when we merge we'll fix the conflicts by using the main branch version.

Reply:
just carious why is it required? The release tag `v1.7.1` will define the version of the new release not the prefix.

Reply:
It might not be necessary for our release pipeline because we force it during build and package steps: https://github.com/OrchardCMS/OrchardCore/blob/main/.github/workflows/release_ci.yml#L40-L42

However it's used when publishing to the preview feed: https://github.com/OrchardCMS/OrchardCore/blob/main/.github/workflows/preview_ci.yml#L30-L31

So maybe we don't need it here. You can try to revert this change and we'll deploy as-is. I believe it should work. If the package version is not correct then it will fail and we can fix. Worst case the assembly file properties is not "totally" correct. Once we know we can add a comment to the file for future reference.

Reply:
reverted. Let me know if you want to merge this one and then merge it back into main. I can release 1.7.1 if you are good with it.